config/tests/testgrids/config_test.go

```diff
@@ -503,6 +503,34 @@ func TestKubernetesProwInstanceJobsMustHaveMatchingTestgridEntries(t *testing.T)
 	}
 }
 
+// WIP See policy description on GH Issue #18599
+func TestJobsMustHaveTestGridAlertsSetup(t *testing.T) {
+	for _, dashboard := range cfg.Dashboards {
+		for _, dashboardtab := range dashboard.DashboardTab {
+			if dashboardtab.GetAlertOptions() == nil {
+				t.Errorf("FAIL : %s : CI Signal Policy Violation: A DashboardTab config MUST CONTAIN an alert contact email address.", dashboardtab.GetName())
+
+			}
+		}
+	}
+}
+
+// WIP subtest version - no real advantage ideally I would have liked subtests
+// to register as counted failures. AFAIK no idiomatic golang test way of
+// getting subtest stats.
```

Comment:
I'm not 100% clear on this either, don't have time to dig in.  However I feel like you could gather stats and log them if you'd like ("x / y failing policy")  I also think test-case-per-dashboard means hundreds or thousands of tests, which probably wouldn't make sense unless each test covered _all_ policies.